app/models/tag.rb

```diff
@@ -6,6 +6,7 @@
 #
 #  id         :bigint           not null, primary key
 #  color      :text
+#  emoji      :string
```

Comment:
Add a validation that it's one emoji and that the text is actually an emoji.

https://domchristie.co.uk/posts/ruby-emoji/